config/routes.rb

```diff
@@ -16,4 +16,7 @@
   resources :players, only: [:index]
 
   get '/profile', to: 'users#show'
+  # patch '/profile', to: 'users#update'
+
+  get '/verification/:user_id', to: 'users#verify'
```

Comment:
What do you guys think about setting this route up to not display the user_id in the url?